### ISSUE TITLE
Number of data points lower than number of chromosomes

### ISSUE THREAD

**Issue by LorenaDerezanin** [Lakshay-Anand/chromoMap.github.io#1]
Hi Lakshay,

thank you for creating the chromoMap, I really like it!

At the moment, I'm trying to plot a set of structural variants (SVs) onto chromosomes. I'm using chromomap pkg v.0.2.
Previously, when I used larger SV sets, I haven't had any issues, 
but this time, the set contains only 10 SVs and 20 chromosomes,
and the chromomap doesn't fully run, but stops and displays an incomplete error message:
```
OUTPUT: 
Number of Chromosome sets: 1 
Number of Chromosomes in set  1 : 19 
Processing data.. 
Number of annotations in data set  1 : 10 
Error in chromoMap("input_data/Chr_elements.txt", "input_data/SV_elements.txt",  :

```
I assumed that chromomap requires equal number of chromosomes in the variant set and in the given chr set, regardless the number of variants, so I attempted to overcome the issue by adding the "empty"
chromosomes (where no SVs have been detected) with zeroes passed as variant coordinates, but this unfortunately didn't trick the chromomap, and the same error was thrown.

What would be your suggestion for solving this issue?
Would it be possible for you to tweek chromomap to enable it to plot on chromosomes when there is lower number of data points then chromosomes like in this case?

Thank you in advance,
Lorena

**Comment by Lakshay-Anand** [comment]
Hi Lorena,

Thank you for reaching out to me and for the compliments, I really
appreciate that.

regarding your issue, there is no such requirement that you must include
all chromosomes in the annotations file. For instance, I have constructed
the following plot that renders 5 chromosomes with only two annotations on
chr 1 and 3 respectively.
![Rplot](https://user-images.githubusercontent.com/37417608/123685193-6a5b8a00-d81c-11eb-9bda-4219d8ca9f8f.png)

Would it possible for you to share the input files so I can have look at
them to determine the cause of this error? Also, since you said that
there are 20 chromosomes while the chromoMap is detecting only 19, I would
need to see your chromosome file as well.

Happy to help!

Regards,
Lakshay

**Comment by LorenaDerezanin** [comment]
Hi Lakshay,

thank you for a quick reply. In the meantime, I updated chromomap to v0.3.

There are indeed 19 chromosomes in the set, detected correctly by chromomap.
I've noticed what caused an error, it was my mistake.
I had 3 colors passsed in a vector as `data_colors` input, as 3 SV types are expected (DEL, DUP, INV), but only 2 different variant types are contained in this data set. So correcting it to two solved the issue and SVs were plotted without any trouble.

Sorry for the confusion and thanks again! 
L
